lighthouse-core/lib/client-rect-functions.js

```diff
@@ -0,0 +1,311 @@
+/**
```

Comment:
Can all instances of `client` be removed? seems this could be generalized to just fun with rectangles.

Reply:
Good point, I've split it into rect helpers and tap target audit specific code now.